src/cargo/ops/cargo_compile/mod.rs

```diff
@@ -643,6 +647,7 @@ fn traverse_and_share(
         unit.is_std,
         new_dep_hash,
         unit.artifact,
+        None,
```

Comment:
Could we have a simple comment explaining why a `None` being passed?

Reply:
Done